src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/ContactDetailsTabMailing.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Link, styled, Typography } from '@material-ui/core';
+import { LocationOn } from '@material-ui/icons';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ContactDetailsTabQuery } from '../ContactDetailsTab.generated';
+
+const ContactDetailsMailingTitle = styled(Typography)(({ theme }) => ({
+  marginTop: theme.spacing(2),
+}));
+
+const ContactDetailsMailingMainContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  marginTop: theme.spacing(2),
+}));
+
+const ContactDetailsMailingIcon = styled(LocationOn)(({ theme }) => ({
+  margin: theme.spacing(1),
+  width: '14px',
+  height: '20px',
+  flexGrow: 1,
+}));
+
+const ContactDetailsMailingTexContainer = styled(Box)(({}) => ({
+  flexGrow: 8,
+}));
+
+const ContactDetailsMailingLabelTextContainer = styled(Box)(({}) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  marginTop: '10px',
+}));
+
+const ContactDetailsMailingLabel = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.hint,
+  marginRight: '5px',
+}));
+
+interface MailingProp {
+  data: ContactDetailsTabQuery;
+}
+
+export const ContactDetailsTabMailing: React.FC<MailingProp> = ({ data }) => {
+  const { t } = useTranslation();
+  return (
+    <Box>
+      <ContactDetailsMailingTitle variant="h6">
+        {t('Mailing')}
+      </ContactDetailsMailingTitle>
+      <ContactDetailsMailingMainContainer>
+        <ContactDetailsMailingIcon />
+        <ContactDetailsMailingTexContainer>
+          {/* Address Section */}
+          <Typography variant="subtitle1">{data.contact.name}</Typography>
+          <Typography variant="subtitle1">
+            {data.contact.primaryAddress.street}
+          </Typography>
+          <Typography variant="subtitle1">
+            {data.contact.primaryAddress.city +
+              ', ' +
+              data.contact.primaryAddress.state +
+              ' ' +
+              data.contact.primaryAddress.postalCode}
```

Comment:
Do we need a loading state for this? Or does this not get rendered until the data is available? Also, you could use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here. A lot better than having concatting the string together.
```suggestion
{`${data.contact.primaryAddress.city}, ${data.contact.primaryAddress.state}, ${data.contact.primaryAddress.postalCode}`}
```
You could also do some [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring) so you don't have to dive so deep into the object to get the value you want for the string. You would probably need something like this.

```
  const {
    contact: {
      primaryAddress: { street, city, state, postalCode },
    },
  } = data;
```
And then all you would have for the string is
```
{`${city}, ${state}, ${postalCode}`}
```